packages/web-app-files/src/components/Collaborators/Collaborator.vue

```diff
@@ -156,7 +156,7 @@
             v-oc-tooltip="editShareHint"
             :aria-label="editShareHint"
             appearance="raw"
-            class="files-collaborators-collaborator-edit"
+            class="files-collaborators-collaborator-edit oc-m-xs"
```

Comment:
Please use `oc-mr-xs` to apply the margin only on the right (oc-`margin right`-`xsmall`)